<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QDialog" name="window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>8000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texto a voz </string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="createFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>10</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear archivo mp3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>50</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reproducir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tutorial">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>6</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="placeholderText">
    <string>Escriba su texto aqu√≠...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del archivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
